Spanner/src/Connection/Grpc.php

```diff
@@ -1361,4 +1366,20 @@ private function getLroResponseMapper($typeUrl)
 
         return null;
     }
+
+    private function setRequestOptions(array &$args)
+    {
+        if (!empty($args['tags'])) {
+            $tags = $args['tags'];
```

Comment:
`RequestOptions` currently contains both tagging and RPC priority options. I'm not sure if it makes more sense to have separate entries (i.e. `tags` and `priority`); or if it we should use `requestOptions` and use the serializer to decode the message. i.e.:
```
$requestOptions = $this->pluck('requestOptions', $args, false) ?: [];
$args['requestOptions'] = $this->serializer->decodeMessage(
    new RequestOptions,
    $requestOptions
);
```
Personally I would prefer the latter as it is easier to maintain.

Is there a reason to prefer not exposing a `requestOptions` option rather than exposing separate `tags` and `priority` options to users?